Exception on /signup [GET]
Traceback (most recent call last):
  File "/home/jyoti/MSDS/MSDS603/MUIMUI/venv/lib/python3.6/site-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/jyoti/MSDS/MSDS603/MUIMUI/venv/lib/python3.6/site-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/jyoti/MSDS/MSDS603/MUIMUI/venv/lib/python3.6/site-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/jyoti/MSDS/MSDS603/MUIMUI/venv/lib/python3.6/site-packages/flask/_compat.py", line 33, in reraise
    raise value
  File "/home/jyoti/MSDS/MSDS603/MUIMUI/venv/lib/python3.6/site-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/jyoti/MSDS/MSDS603/MUIMUI/venv/lib/python3.6/site-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/jyoti/MSDS/MSDS603/MUIMUI/src/webapp/app/views/main.py", line 68, in signup
    return render_template('/user/signup')
  File "/home/jyoti/MSDS/MSDS603/MUIMUI/venv/lib/python3.6/site-packages/flask/templating.py", line 127, in render_template
    return _render(ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/home/jyoti/MSDS/MSDS603/MUIMUI/venv/lib/python3.6/site-packages/jinja2/environment.py", line 869, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/home/jyoti/MSDS/MSDS603/MUIMUI/venv/lib/python3.6/site-packages/jinja2/environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "/home/jyoti/MSDS/MSDS603/MUIMUI/venv/lib/python3.6/site-packages/jinja2/environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "/home/jyoti/MSDS/MSDS603/MUIMUI/venv/lib/python3.6/site-packages/jinja2/loaders.py", line 113, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "/home/jyoti/MSDS/MSDS603/MUIMUI/venv/lib/python3.6/site-packages/flask/templating.py", line 64, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: /user/signup
